content/id/docs/tasks/administer-cluster/highly-available-master.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Mengatur Master Kubernetes dengan Ketersediaan Tinggi (High-Availability)
+content_type: task
+---
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.5" state="alpha" >}}
+
+Kamu dapat mereplikasi master Kubernetes dalam skrip `kube-up` atau `kube-down` untuk Google Compute Engine (GCE).
+Dokumen ini menjelaskan cara menggunakan skrip kube-up/down untuk mengelola master dengan ketersedian tinggi atau _high_availability_ (HA) dan bagaimana master HA diimplementasikan untuk digunakan dalam GCE.
+
+
+
+## {{% heading "prerequisites" %}}
+
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+
+
+<!-- steps -->
+
+## Memulai klaster yang kompatibel dengan HA
```

Comment:
oh nice I learned kompatibel is indonesian word https://kbbi.kemdikbud.go.id/entri/kompatibel